packages/client/src/PhantomClient.ts

```diff
@@ -196,6 +200,30 @@ export class PhantomClient {
     }
   }
 
+  private async prepare(
+    transaction: string,
+    organizationId: string,
+    submissionConfig: SubmissionConfig,
+    account: string,
+  ): Promise<AugmentWithSpendingLimitResponse> {
```

Comment:
I would call this with a simpler name
```suggestion
  ): Promise<PrepareResponse> {
```